src/plugins/auctionHouseModule/Bid.ts

```diff
@@ -42,9 +43,11 @@ export type Bid = Readonly<
     canceledAt: Option<DateTime>;
   } & (
     | {
-        token: TokenWithMetadata;
+        isPublic: false;
+        token: TokenWithMetadata | TokenWithMint;
```

Comment:
Could you please revert that change? In Auction House, one cannot place a bid on a token or a mint account that is not associated with a metadata account. In order words, the metadata account is mandatory so I'd like the type to reflect that.
```suggestion
        token: TokenWithMetadata;
```

Reply:
I will just add a throw in case it's TokenWithMint

Reply:
Gotcha, will update